core/shared/src/main/scala/zio/prelude/ZValidation.scala

```diff
@@ -355,9 +372,18 @@ sealed trait ZValidation[+W, +E, +A] { self =>
 object ZValidation extends LowPriorityValidationImplicits {
 
   final case class Failure[+W, +E](log: Chunk[W], errors: NonEmptyChunk[E]) extends ZValidation[W, E, Nothing] {
-    lazy val errorsUnordered: NonEmptyMultiSet[E] = NonEmptyMultiSet.fromIterable(errors.head, errors.tail)
+    lazy val errorsUnordered: NonEmptyMultiSet[E]                          = NonEmptyMultiSet.fromIterable(errors.head, errors.tail)
+    def toException(implicit ev: E <:< Throwable): Failure.Exception[W, E] = Failure.Exception(this)
+  }
+
+  object Failure {
+    final case class Exception[+W, +E](failure: Failure[W, E])(implicit ev: E <:< Throwable)
+        extends scala.Exception(ev(failure.errors.head)) {
```

Comment:
WDYT of:
```suggestion
        extends RuntimeException(ev(failure.errors.head)) {
```

Why are you using `scala.Exception`? First time I see this TBH 🤔

Reply:
Reworked